src/shared/ui/select/select-context.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react'
+
+export interface SelectContextType {
+    isOpen: boolean
+    toggleOpen: () => void
+    selectedValue: string | null
+    selectValue: (value: string) => void
```

Comment:
변수명이 통일 안된느낌이 있는것같아요!
selectedValue는 변수고 selectVlaue는 함순데 비슷해서 타입을 안보면 뭐가 변수고 함수지? 하는 느낌이 있습니다
제가 잘 짓는 건 아니라 동현님이 변수명을 추천해주실거라 믿습니다

Reply:
```suggestion
    isOpen: boolean
    toggle: () => void
    selectedValue: string | null
    onChangeValue : (value: string) => void
```
toggle은 열고 닫는걸 의미하는데 toggleOpen은 열고닫고 연다? 라는 느낌일것 같아요. 

selectValue와 selectedValue도 결국은 같은 이름이라서 onSelectedValue로 변경하는게 좋아보입니다.

나중에 네이밍 지으실 때, 내부에서 관리하는 상태인지, 외부에서 사용하는 건지 맥락에 따라 네이밍도 고려해주시면 좋을것 같아요. 내부에서 사용하는 용도라면 상태로, 외부에서 사용하는거라면 선언적(What)으로 작성하는게 좋습니다.